app/build.gradle.kts

```diff
@@ -124,6 +125,9 @@ dependencies {
     // Settings Screen
     implementation(project(":feature:settings"))
 
+    // Domain Camera
+    implementation(project(":domain:camera"))
```

Comment:
I'm not sure if the app should have a direct dependency on `domain:camera`. Maybe we need a layer of abstraction? @yasith do you know?

Reply:
ImageCaptureEvent can be moved into feature:preview to prevent this line of change